### ISSUE TITLE
Docker 2.1.5.0 not accessible from WSL 2

### ISSUE THREAD

**Issue by ad-on-is** [docker/for-win#5061]
I just updated to Docker Desktop 2.1.5.0

On the previous version, when I started Docker with the WSL 2 Tech preview, I was able to access it from Ubuntu inside WSL 2, now this doesn't work anymore.

```
add@ADISDELL $ docker ps
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

![image](https://user-images.githubusercontent.com/9553253/68199195-d50fe000-ffbd-11e9-8431-07353a8adb05.png)

![image](https://user-images.githubusercontent.com/9553253/68199201-dccf8480-ffbd-11e9-8d84-53e299a4eb29.png)

I also get this error message, when I restart docker.

![image](https://user-images.githubusercontent.com/9553253/68199257-fa045300-ffbd-11e9-96b6-cf15c95cfd18.png)

## Solved
I was able to solve the problem, by entering the shown command-line in the dialog, in an elevated PowerShell.

**Comment by benjlevesque** [comment]
Thanks @ad-on-is, I had the same problem and your issue helped me fix it!

The recommended command wasn't enough (though it might be necessary, unclear). 
After that I had issues starting docker which was fixed by https://github.com/docker/for-win/issues/4393#issuecomment-541780066. 

Then, I could open Docker Dashboard (tray Icon > Dashboard), go to Settings > Resources > WSL integration, and enable the distro I wanted (as shown in your screenshot)

**Comment by keckelt** [comment]
My (only) one linux distro was also deactivated after updating (and I didn't know of/ find the checkbox in the submenu).

**Comment by seayeo1066** [comment]
I had enabled the distro as shown in the OP screenshot but still could not connect to the Docker daemon.  (I had no errors about WSL2 not being installed and could happily do no-docker things in a WSL2 terminal.) Removing the Windows Security CFG override as described in the issue linked by @benjlevesque resolved my issue. I did not need to start vmcompute as it was already running. Just removing the override was enough.

**Comment by Silic0nS0ldier** [comment]
I encountered the issue described here but the solution the OP landed on did not work. Notably, the WSL integration section under resources (and some other sections) where missing after updating from the last WSL2 tech preview. Reinstalling Docker for Windows Edge however did resolve the problem for me. The Docker WSL distributions were also notably missing, so evidently something went horribly wrong during the update (this is a preview, so not too much of a surprise).

Only thing I had to do afterwards to use the Docker WSL2 backend within a WSL2 image afterwards was install the Docker CLI, tick my distro under Resources -> WSL Intregration and finally add the distro user to the Docker group.
